test/plugin/test_out_forward.rb

```diff
@@ -1294,33 +1288,28 @@ def plugin_id_for_test?
         keepalive_timeout 2
       ]
       d = create_driver(output_conf)
-      d.instance_start
 
-      begin
-        chunk = Fluent::Plugin::Buffer::MemoryChunk.new(Fluent::Plugin::Buffer::Metadata.new(nil, nil, nil))
-        mock.proxy(d.instance).socket_create_tcp(TARGET_HOST, @target_port,
-                                                 linger_timeout: anything,
-                                                 send_timeout: anything,
-                                                 recv_timeout: anything,
-                                                 connect_timeout: anything
-                                                ) { |sock| mock(sock).close.once; sock }.once
+      chunk = Fluent::Plugin::Buffer::MemoryChunk.new(Fluent::Plugin::Buffer::Metadata.new(nil, nil, nil))
+      mock.proxy(d.instance).socket_create_tcp(TARGET_HOST, @target_port,
+                                               linger_timeout: anything,
+                                               send_timeout: anything,
+                                               recv_timeout: anything,
+                                               connect_timeout: anything
+                                              ) { |sock| mock(sock).close.once; sock }.once
 
-        target_input_driver.run(timeout: 15) do
-          d.run(shutdown: false) do
-            node = d.instance.nodes.first
-            2.times do
-              node.send_data('test', chunk) rescue nil
-            end
+      target_input_driver.run(timeout: 15) do
+        d.run do
```

Comment:
In this case, `d` will be terminated automatically when the block is finished since the default value of `shutdown` is `true`.

Reply:
?
So I have removed `d.shutdown` and `shutdown: false`.
Something wrong?

Reply:
I understand!
It is comparing to https://github.com/fluent/fluentd/pull/4433#discussion_r1521403776 !
Sorry for confusing.

https://github.com/fluent/fluentd/pull/4433#discussion_r1521403776 is wrong.

Reply:
Yup, it's just a memo. I wrongly submitted this comment before submitting https://github.com/fluent/fluentd/pull/4433#discussion_r1521403776. Sorry for confusing.